README.md

```diff
@@ -433,6 +428,48 @@ func LogResponse() autorest.RespondDecorator {
 }
 ```
 
+## Tracing and Metrics
+
+All packages and the runtime are instrumented using [OpenCensus](https://opencensus.io/).
+
+### Enable
+
+As of now, tracing is disabled by default. To enable it:
```

Comment:
I would slightly reword the second sentence to make it clear these are mutually exclusive.  Something like "There are two ways to enable tracing:"